protected/controllers/adminFile/DeleteFileAttributeAction.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * This action will delete file attributes in admin file update page
+ */
+
+class DeleteFileAttributeAction extends CAction
+{
+    public function run()
+    {
+        if (!Yii::app()->request->isPostRequest)
+            throw new CHttpException(404, "The requested page does not exist.");
+
+        if (isset($_POST['id'])) {
+            $attribute = FileAttributes::model()->findByPk($_POST['id']);
+
+            if ($attribute) {
+                $file = new File;
+                $file->id = $attribute->file_id;
+                $out = print_r($file->dataset_id, true);
+                file_put_contents('fileobj.log', $out);
```

Comment:
@pli888 I want to get `dataset_id` from  table `file` and then pass it to `createCurationLogEntry`,  but nothing is returned when I check the `fileobj.log`. Can you have a look for me?

Reply:
I don't think you need to be creating a new `File` object on line 17 to get hold of the `dataset_id`. You should be able to get it by querying through the properties of objects starting from `$attribute` using `$out = $attribute->file->dataset_id;`